Dockerfile.s3-scality

```diff
@@ -1,29 +1,24 @@
-FROM node:6-slim
+FROM node:8-slim
 
+ENV NO_PROXY localhost,127.0.0.1
+ENV no_proxy localhost,127.0.0.1
 
-WORKDIR /usr/src/app
+EXPOSE 8000
+
+COPY ./package.json ./package-lock.json /usr/src/app/
 
-# Keep the .git directory in order to properly report version
-COPY ./package.json .
+WORKDIR /usr/src/app
 
 RUN apt-get update \
-    && apt-get install -y jq python git build-essential --no-install-recommends \
+    && apt-get install -y jq python git curl build-essential --no-install-recommends \
     && npm install --production \
-    && apt-get autoremove --purge -y python git build-essential \
-    && rm -rf /var/lib/apt/lists/* \
-    && npm cache clear \
     && rm -rf ~/.node-gyp \
-    && rm -rf /tmp/npm-* \
-    && apt-get clean
+    && rm -rf /tmp/npm-*
```

Comment:
you removed that 'autoremove' and apt-get clean. It will increase the image size